store/secretsmanagerstore.go

```diff
@@ -0,0 +1,562 @@
+package store
```

Comment:
Last thing, I promise :)

Would you be willing to be the maintainer of this backend for the time-being? Nothing formal or enforceable obviously. All PRs would still go through Segmenters for approval (we haven't quite figured out how to deal with giving external folks merge access). We'll just defer to you on issues and PRs.

If that sounds good, just add a comment to the top of this file:

```suggestion
// Secrets Manager Store is maintained by Dan McTough https://github.com/danmactough. Thanks Dan!
package store
```

Reply:
No problem. I'll just correct the spelling of my name. 🙃

Reply:
Added via 53be3ab @nickatsegment

Reply:
> No problem. I'll just correct the spelling of my name. 🙃

Whoops! My bad, sorry.